### ISSUE TITLE
CRT Horizontal image adjust

### ISSUE THREAD

**Issue by mabrilG** [MiSTer-devel/TurboGrafx16_MiSTer#108]
The core is in an almost perfect state. This is a minor detail, but the horizontal centering of the image does not match the original hardware. I put two screenshots to see the difference, it's about 8 pixels.

Thank you very much for the work done.

Original hardware:
![OriginalHardware](https://user-images.githubusercontent.com/38905302/83788554-ae21a580-a695-11ea-8fed-5d041e3f9aa2.jpg)
Fpga core:
![FpgaCore](https://user-images.githubusercontent.com/38905302/83788592-b974d100-a695-11ea-9a15-9d2ac3e79939.jpg)

**Comment by mike2love** [comment]
I have noticed the same thing since the last core update. Was not there before.

**Comment by sorgelig** [comment]
i've adjusted the position in new release.
I have no dummy CRT. I only have external VGA->HDMI scaler which doing some automatic adjustments. So not sure if new position is correct.

**Comment by mabrilG** [comment]
Image position is almost the same now that original hardware, the difference is negligible. Incredible that you have adjusted so much without CRT.
Thanks!.
I close the issue.

**Comment by sorgelig** [comment]
can you post comparison of new build with real HW? I will fine tune it in next build

**Comment by mabrilG** [comment]
Hello Sorgelig. I have done some tests changing the value of the constant HS_OFF in the file huc6260vhd. The value I've found that center the image exactly the same as the original hardware is 46. I hope I have helped.
I show you how it looks, no difference is appreciated.
Original Hardware:
![OriginalHardware](https://user-images.githubusercontent.com/38905302/83918834-ce2a9500-a779-11ea-9f17-24ddfa598ff0.jpg)
Fpga Core:
![FpgaCore](https://user-images.githubusercontent.com/38905302/83918854-d4b90c80-a779-11ea-924c-707693856674.jpg)
